### PR TITLE
Remove use of $GLOBALS['HTTP_RAW_POST_DATA']

### LINKED ISSUE #10: $GLOBALS['HTTP_RAW_POST_DATA'] is deprecated in PHP 5.6, from AbstractCallback.php
The [AbstractCallback](https://github.com/zendframework/zend-feed/blob/5143d54b20d9dbab1a73179a89b2a1ff0e70e7d2/src/PubSubHubbub/AbstractCallback.php#L275-L290) class makes use of `$GLOBALS['HTTP_RAW_POST_DATA']`, which is deprecated in PHP 5.6.

``` php
    /**
     * Return the raw body of the request
     *
     * @return string|false Raw body, or false if not present
     */
    protected function _getRawBody()
    {
        $body = file_get_contents('php://input');
        if (strlen(trim($body)) == 0 && isset($GLOBALS['HTTP_RAW_POST_DATA'])) {
            $body = $GLOBALS['HTTP_RAW_POST_DATA'];
        }
        if (strlen(trim($body)) > 0) {
            return $body;
        }
        return false;
    }
```

I suggest either:
1. Removing those three lines entirely, as it will not be automatically filled in the future
2. Checking for PHP < 5.6 beforehand

### PR DESCRIPTION
**Pull request by RobLoach**
Fixes #10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": ">=5.6",
```
Comment (samsonasik) [inline composer.json]: this should be reverted.

Comment (RobLoach) [inline composer.json]: Ah, reading quickly through http://php.net/manual/en/wrappers.php.php#wrappers.php.input , I thought it was not available in <= PHP 5.5. I see I was mistaken. Thanks! I'll update.

**Comment by Maks3w** [comment]
Note: Prior to PHP 5.6, a stream opened with php://input could only be read once

**Comment by Xerkus** [comment]
This change is a go now that minimum php version is bumped to 5.6

@RobLoach would you rebase and update tests that are dependent on `HTTP_RAW_POST_DATA`?

**Comment by weierophinney** [comment]
Closing in favor of #74, which builds on this one, but updates the tests accordingly as well.

**Pull request closed without merging**